src/qt/splashscreenveil.cpp

```diff
@@ -49,7 +49,7 @@ SplashScreenVeil::SplashScreenVeil(interfaces::Node& node, Qt::WindowFlags f, co
     this->setStyleSheet(GUIUtil::loadStyleSheet());
 
     this->resize(800, 514);
-    ui->frame->setStyleSheet("#frame{border-image: url(\":/icons/splash_background\") 0 0 0 0 stretch stretch;padding: 0;margin: 0;}");
+    //ui->frame->setStyleSheet("#frame{border-image: url(\":/icons/splash_background\") 0 0 0 0 stretch stretch;padding: 0;margin: 0;}");
```

Comment:
Why?

Reply:
Because we are setting the stylesheet here and the whole purpose is the stylishness are now in stylishness instead of being handled in the code. It should have a matching styling in the css files.